### ISSUE TITLE
Using Auth0 for Authentication

### ISSUE THREAD

**Issue by harshsvaidya** [harsh432004/attendance-system-app#2]
Let's use Auth0 for authentication instead of streamlit auth

**Comment by Frpratik** [comment]
i can do this. could you please assign this task to me?

**Comment by harshsvaidya** [comment]
Yes Assigned, please go ahead and let me know if you have any doubts!

**Comment by Frpratik** [comment]
you havent assigned this to me yet,have you?

**Comment by harshsvaidya** [comment]
yes, I have already assigned this issue to you please check

**Comment by harshsvaidya** [comment]
Please check it, i have assigned you again

**Comment by TheChaoticor** [comment]
@Frpratik any updates?

**Comment by Frpratik** [comment]
In progress.

**Comment by TheChaoticor** [comment]
Any updates?

**Comment by harshvaidya04** [comment]
@Frpratik any updates on this issue?

**Comment by Frpratik** [comment]
Hi brother. I have done with this issue but I don't know why your code is not working perfectly. Let me fix this in two days and let you know.

**Comment by TheChaoticor** [comment]
ok

**Comment by Frpratik** [comment]
I have done with this and added pull request. Please check it out and let know if you need any help or my assistance.
